### PR TITLE
Lrn 44065/feature/authoraide node sdk

### PR DESCRIPTION
**Pull request by sreenivasa-murty-lrn**
Implement the standalone authoraide api for nodesdk

### CODE REVIEW

**Review by markus-liang-lrn — APPROVED** [review]
lgtm

File: docs/quickstart/views/main.ejs [hunk 1]
```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html>
+<head lang="en">
+    <meta charset="UTF-8">
+    <title>Learnosity SDK - NodeJS</title>
+    <script src="//authoraide.dev.learnosity.com"></script>
```
Comment (bhavya-shukla-lrn) [inline docs/quickstart/views/main.ejs]: I think this should be as below even though it has CORS issue. but lets discuss this tomorrow in standup
```suggestion
    <script src="//authoraide.learnosity.com"></script>
```

**Review by bhavya-shukla-lrn — APPROVED** [review]
LGTM

File: docs/quickstart/views/main.ejs [hunk 2]
```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html>
+<head lang="en">
+    <meta charset="UTF-8">
+    <title>Learnosity SDK - NodeJS</title>
+    <script src="//authoraide.learnosity.com"></script>
+</head>
+<body>
+    <h1>Standalone Authoraide API Example</h1>
+<!-- We create a div element with the id learnosity-author into which the Author API will load -->
```
Comment (markus-liang-lrn) [inline docs/quickstart/views/main.ejs]: Author API -> Authoraide API

**Comment by sreenivasa-murty-lrn** [comment]
Merged to master

**Pull request merged**